inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -102,7 +95,11 @@ bool isCMXEnough(int cmxSize, int numSlices, std::vector<int> bufferSizes) {
         return true;
     };
 
-    return std::all_of(bufferSizes.begin(), bufferSizes.end(), buffer_allocate);
+    return (numSlices > 0) && std::all_of(bufferSizes.begin(), bufferSizes.end(), buffer_allocate);
```

Comment:
What's happening when numSlices == 0?

Reply:
the function can return `true`, but should return `false`

Reply:
Thanks